archinstall/lib/installer.py

```diff
@@ -623,6 +623,49 @@ def add_bootloader(self, bootloader='systemd-bootctl'):
 				SysCommand(f'/usr/bin/arch-chroot {self.target} grub-install --target=i386-pc --recheck {boot_partition.path}')
 				SysCommand(f'/usr/bin/arch-chroot {self.target} grub-mkconfig -o /boot/grub/grub.cfg')
 				self.helper_flags['bootloader'] = True
+
+		elif bootloader == 'efistub':
+			self.pacstrap('efibootmgr')
+
+			if not has_uefi():
+				raise HardwareIncompatibilityError
+			# TODO: Ideally we would want to check if another config
+			# points towards the same disk and/or partition.
+			# And in which case we should do some clean up.
+
+			for kernel in self.kernels:
+				# Setup the firmware entry
+
+				label = f'Arch Linux ({kernel})'
+				loader = f"/vmlinuz-{kernel}"
+
+				kernel_parameters = []
+
+				if not is_vm():
+					vendor = cpu_vendor()
+					if vendor == "AuthenticAMD":
+						kernel_parameters.append("initrd=\\amd-ucode.img")
+					elif vendor == "GenuineIntel":
+						kernel_parameters.append("initrd=\\intel-ucode.img")
+					else:
+						self.log("unknow cpu vendor, not adding ucode to systemd-boot config")
+
```

Comment:
This message references systemd-boot but this is efistub and doesn't use a bootloader

Reply:
Good catch, changed in 436cabb8d057d88f97fc7f51ba1d5abd4a6b7fd7